book/src/requirements.md

```diff
@@ -0,0 +1,166 @@
+# Requirements
+
+The design of Orchard aims to be mostly identical to the Sapling circuit in functionality.
+
+## Functional Protocol Requirements
+
+Functional protocol requirements aim to objectively define what the protocol achieves in a manner that can be verified by any reviewer with sufficient domain expertise.
+
+Functional protocol requirements are scoped to the protocol design itself, which excludes important broader considerations for Orchard as a whole, such as usability goals for a wallet which aren't directly constrained by the protocol.
+
+Finally, there is a category of proposed requirements that could have been coherently in-scope for Orchard which were excluded due to time & complexity constraints.
+
+These scope clarifications exlude some draft requirements found in earlier revisions of this document, in pull request comments, or elsewhere. Many of those could become ZIPs in their own right.
+
+### ProtoR1. Orchard Relies on a Zero-Knowledge Proving System that is Secure in The Uniform Random String Model
+
+**Requirement:** The Orchard shielded transfer circuit uses a Zero-Knowledge Proving System that is secure in the Uniform Random String (URS) Model, and therefore has no reliance on a trusted setup.
+
+**Rationale - UX:** Users who rely on Orchard are not directly vulnerable to the class of trusted parameter compromises.
+
+**Rationale - Strategic:** As Sprout and Sapling usage declines or are deprecated, the ZEC a whole will become protected from any previous trusted setup compromise henceforth.
+
+**Rationale - Strategic:** By avoiding a requirement on a trusted setup, Orchard *and future* upgrades no longer need a parameter setup MPC process, which improves protocol agility and the ability to respond quickly to security vulnerabilities.
+
+### ProtoR2. Orchard Addresses
+
+**Requirement:** Orchard introduces a new address format with an unambiguous user-facing encoding.
+
+**Rationale - UX:** A new address format allows users to identify which ZKP platform their funds rely on.
+
+**Rationale - UX:** A new address format reinforces the fact that users must migrate funds to opt into Orchard.
+
+**Rationale - UX:** A new address format, in combination with the turnstile design, signals to users a boundary for turnstile-based supply integrity protections.
+
+**Rationale - UX:** A new address format, in combination with the turnstile design, signals to users a boundary for privacy-guarantees.
+
+**Counter-rationale - UX:** This complicates the simplified narrative of "z→z has strong privacy" because migrating between pools has a privacy cost.
```

Comment:
There shouldn't be any reference to [ZIP 308](https://zips.z.cash/zip-0308) here, if that's what you meant @nuttycom, because we don't intend to use it for the Sapling -> Orchard migration.